ui/src/app/login/login.tsx

```diff
@@ -25,8 +26,9 @@ function getRedirect(): string {
 
 export function Login() {
     useCollectEvent('openedLogin');
+    const { t } = useTranslation();
```

Comment:
```suggestion
    const { t } = useTranslation();
    
    useCollectEvent('openedLogin');

```


This would follow the conventions we have elsewhere -- variable declarations at the top, new line, effect, new line, render.